EDDI/EDDI.cs

```diff
@@ -1293,6 +1300,40 @@ private bool eventVehicleDestroyed(VehicleDestroyedEvent theEvent)
             return true;
         }
 
+        private bool eventStatus(StatusEvent theEvent)
```

Comment:
Not a review comment but a note to self: this code feels very weighed down. Multilple calls to `Instance` are a code smell for both too much coupling and too little cohesion.